t04_refactored.py

```diff
@@ -119,10 +119,40 @@ def scott_adventure(username, delay):
 ###################################################################################
 ###################################################################################
 
-def team_1_adv(username, delay):
+def team_1_adv(delay = 1.0, username = "lakesd3_oludairoa"):
+ #https://docs.google.com/document/d/1VpipMYDS0OLnxJ6RpZfe3CgiNNkQeHDKCK7lMajeUqU/edit?usp=sharing
+ # Dawson Lakes
+ # Ayomide Oludairo
+ #
     pass
-    # TODO Add your code here
-    return True
+
+    delay = 1.0
+    isDead = False
+
+    username = input("What is your name?")
```

Comment:
You don't need to ask for the username anymore, since it's an input parameter also.